library/modules/Maps.cpp

```diff
@@ -118,8 +117,24 @@ void Maps::getSize (uint32_t& x, uint32_t& y, uint32_t& z)
     y = world->map.y_count_block;
     z = world->map.z_count_block;
 }
+void Maps::getSize (uint32_t& x, uint32_t& y, uint32_t& z)
+{
+    if (!IsValid())
+    {
+        x = y = z = 0;
+        return;
+    }
+    x = int32_t(world->map.x_count_block);
+    y = int32_t(world->map.y_count_block);
+    z = int32_t(world->map.z_count_block);
+}
 
 // getter for map size in tiles
+void Maps::getTileSize (int32_t& x, int32_t& y, int32_t& z){
+    getSize(x, y, z);
+    x *= 16;
+    y *= 16;
+}
```

Comment:
These take references, `ReadFeature` does not ie. you can't cast (I'd say something about rvalues or w/e but I struggle with those concepts). The existing methods should simply be removed because they are incorrect, but I'm trying to patch problems not create more. If you could tell me how to properly deprecate them that'd be great

Reply:
I think I'd do something like this:
```
void Maps::getSize (uint32_t& x, uint32_t& y, uint32_t& z)
{
    int32_t intx, inty, intz:
    getSize(intx, inty, intz);
    x = intx; y = inty; z = intz;
}
```

Reply:
I get the inclination but why let the compiler add a second call? If it doesn't inline the function in the other it's that extra overhead for no good reason

Reply:
Like I commented in the other, I did this. But the inner call is defined as inline inside the .cpp file so hopefully it gets inlined